xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
remove the private keyword for ctors here and the friend class, no reason to hack headers to prevent programmer errors. if you want to prevent others from creating instances, put it in a comment